libs/dh/metering-point/shared/ui-util/src/lib/dh-yes-no.pipe.ts

```diff
@@ -37,8 +40,6 @@ export class DhYesNoPipe implements PipeTransform {
     return this.transloco.translate('yes');
   }
 
-  constructor(private transloco: TranslocoService) {}
```

Comment:
praise: nice change. Makes it easier to test by using `runInInjectionContext` instead of mocking the class.